lib/ui/settings/interactor/settings_bloc.dart

```diff
@@ -42,6 +47,16 @@ class SettingsBloc extends Bloc<SettingsEvent, SettingsState> {
     emit(state.copyWith(showSecurityNotification: false));
   }
 
+  FutureOr<void> _onDeleteAccountTapped(_OnDeleteAccountTapped event, Emitter<SettingsState> emit) async {
+    emit(state.copyWith(pageState: PageState.loading));

```

Comment:
this should be a button loading and not the entire page loading.  Not a blocker tho

Reply:
I think it should be page loading since it's basically destroying the entire user, and logging out?

But yea, could also be button loading.